sycl/test/reduction/reduction_usm.cpp

```diff
@@ -1,12 +1,16 @@
 // UNSUPPORTED: cuda
 // Reductions use work-group builtins not yet supported by CUDA.
 
+// UNSUPPORTED: linux
```

Comment:
For bugs it is better to use XFAIL instead of UNSUPPORTED because it will XPASS once driver upgrade happens and we do not forget to turn it on.